File: Python/3-Basic-Actions/2-Wait-Explicit.py
from selenium import webdriver
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC

driver = webdriver.Chrome()

try:
    driver.get('https://starandread.com/store/namebook')
    my_element = WebDriverWait(driver, 30).until(EC.presence_of_element_located(('id', 'select-product')))
    print(my_element.text)
finally:
    driver.quit()

File: Javascript/3-Basic-Actions/2-Wait-Explicit.js
const {Builder, By} = require('selenium-webdriver');

(async function(){
    const driver = new Builder().forBrowser('chrome').build();

    try {
        const documentInitialised = () => driver.executeScript('return initialised');

        (await driver).get("https://starandread.com/store/namebook")
        
        (await driver).wait(() => documentInitialised(), 30000);

        let my_element = (await driver).findElement(By.id('select-product'));
        console.log(await my_element.getText());

    } finally {
        (await driver).quit();
    }

})();